app/controllers/user_filters_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
     active_filter_options = params.require(:activeFilterOptions).collect do |filter|
       {
         filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :allowRange, options: []),
-        value: filter.permit(:value)[:value] || filter.require(:value) || false,
```

Comment:
So this seems to specifically occur when you select a relative date filter that does not have custom option such as "today", "yesterday", or "tomorrow". This was a result of the refactor of relative dates in the Advanced Filter such that [this line](https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/public_health/query/AdvancedFilter.js#L413) leaves the value as null in these cases.

A `value` is technically required for all other kinds of filters - so do we just wanna say it's not required at all across the board or do we want to specifically check if it's required depending on the filter type? OR do we actually want to give it a better value in the the line linked above? I feel like it's be more accurate to store the actual values of today/yesterday/tomorrow. @sgober thoughts?

Reply:
I agree that it would make sense to store "today", "yesterday", "tomorrow" in `value` as well

Reply:
I disagree mainly because that's against the convention we have.  We have specific number, date and relative date options that determine what is shown and hidden.  That would be a big effort to overhaul that so for the sake of time I think this is the solution do go with.  But I think it's a bandaid for sure